app/arrays.js

```diff
@@ -3,57 +3,115 @@ exports = typeof window === 'undefined' ? global : window;
 exports.arraysAnswers = {
   indexOf: function(arr, item) {
 
+    var result = -1
+
+      for (var i = 0; i < arr.length; i++) {
+        if (item === arr[i]) {
+          result = i;
+        }
+      }
+    return result
   },
 
   sum: function(arr) {
-
+      return arr.reduce(function(sum, current){
+        return sum + current;
+      }, 0);
   },
 
   remove: function(arr, item) {
 
+    const result = arr.filter(function(value) {
+      return value !== item;
+    })
+    return result
   },
 
   removeWithoutCopy: function(arr, item) {
 
+    for (var i = 0; i < arr.length; i++) {
```

Comment:
you can also use arr.indexOf(item) to find out at what index is the item you need to remove and use it in splice